etc/db_schema.xml

```diff
@@ -0,0 +1,109 @@
+    <constraint xsi:type="primary" referenceId="PRIMARY">
+      <column name="entity_id"/>
+    </constraint>
+    <index referenceId="ADYEN_NOTIFICATION_PSPREFERENCE" indexType="btree">
+      <column name="pspreference"/>
+    </index>
+    <index referenceId="ADYEN_NOTIFICATION_EVENT_CODE" indexType="btree">
+      <column name="event_code"/>
+    </index>
+    <index referenceId="ADYEN_NOTIFICATION_PSPREFERENCE_EVENT_CODE" indexType="btree">
+      <column name="pspreference"/>
+      <column name="event_code"/>
+    </index>
+    <index referenceId="ADYEN_NOTIFICATION_MERCHANT_REFERENCE_EVENT_CODE" indexType="btree">
+      <column name="merchant_reference"/>
+      <column name="event_code"/>
+    </index>
+  </table>
+  <table name="adyen_invoice" resource="default" engine="innodb" comment="Adyen Invoice">
+    <column xsi:type="int" name="entity_id" padding="10" unsigned="true" nullable="false" identity="true" comment="Adyen Invoice Entity ID"/>
+    <column xsi:type="varchar" name="pspreference" nullable="false" length="255" comment="Adyen pspreference of the capture"/>
+    <column xsi:type="varchar" name="original_reference" nullable="true" length="255" comment="Adyen OriginalReference of the payment"/>
+    <column xsi:type="varchar" name="acquirer_reference" nullable="true" length="255" comment="Adyen AcquirerReference of the capture"/>
+    <column xsi:type="int" name="invoice_id" padding="10" unsigned="true" nullable="true" identity="false" comment="Link to Magento Invoice table"/>
+    <column xsi:type="timestamp" name="created_at" on_update="false" nullable="true" default="CURRENT_TIMESTAMP" comment="Created at"/>
+    <column xsi:type="timestamp" name="updated_at" on_update="true" nullable="true" default="CURRENT_TIMESTAMP" comment="Updated at"/>
+    <column xsi:type="decimal" name="amount" scale="4" precision="12" unsigned="true" nullable="true" comment="Field to determine the capture amount."/>
+    <column xsi:type="text" name="status" nullable="true" comment="Field to determine the status of the adyen_invoice"/>
+    <column xsi:type="int" name="adyen_order_payment_id" padding="10" unsigned="true" nullable="true" identity="false" comment="Field to link this row to the an adyen_order_payment row."/>
+    <constraint xsi:type="primary" referenceId="PRIMARY">
+      <column name="entity_id"/>
+    </constraint>
+    <constraint xsi:type="foreign" referenceId="ADYEN_INVOICE_INVOICE_ID_SALES_INVOICE_ENTITY_ID" table="adyen_invoice" column="invoice_id" referenceTable="sales_invoice" referenceColumn="entity_id" onDelete="CASCADE"/>
+    <constraint xsi:type="foreign" referenceId="ADYEN_INVOICE_ADYEN_ORDER_PAYMENT_ID_ADYEN_ORDER_PAYMENT_ENTT_ID" table="adyen_invoice" column="adyen_order_payment_id" referenceTable="adyen_order_payment" referenceColumn="entity_id" onDelete="CASCADE"/>
+  </table>
+  <table name="adyen_state_data" resource="default" engine="innodb" comment="Adyen Payment State Data">
```

Comment:
I think the `adyen_state_data` can be completely dropped since we are not storing the state data in the database anymore